site/_posts/2021-09-16-how-to-authenticate-your-react-app.md

```diff
@@ -0,0 +1,168 @@
+
+For the next step, we will link a user to your brand-new FusionAuth app, ReactAuth, via the dashboard. This is where another useful feature of FusionAuth comes into play, as a 'user data store', aka a centralized location where you can manage all the users of your app in a friendly UI environment. Of course, you can register users from your app if you want to build that functionality in later on, but that is beyond this tutorial's scope.  
+
+* Select `users` from the left-hand side menu.
+* Select `add user`.
+* Give the user a fake email and deselect `send email to setup password`.
+* Give the user a simple password. May I suggest `password`?
+* Click the `save` icon on the top right.
+* Go back to `users`.
+* Select `manage` under the `action` tab for your new user.
+* Select `add registration`. In the dropdown menu, select `React Auth`. If you don't see it, double check that you remembered to push `save` when you created your application.
+
+Well done! You now have at least one user that is linked to your `React Auth` app. Not bad for your app being in development.
+
+## Setting up FusionAuth using Kickstart
+
+Instead of manually setting up FusionAuth using the admin UI, you can also use Kickstart. This lets you get going quickly if you have a fresh installation of FusionAuth. Learn more about how to use [Kickstart](/docs/v1/tech/installation-guide/kickstart/). Here's an example [Kickstart file](https://github.com/FusionAuth/fusionauth-example-kickstart/blob/master/example-apps/path-to-kickstart-here) which sets up FusionAuth for this tutorial.
+
+You'll need to edit that Kickstart file and update the application name and redirect URLs.
+
+## Write the React Application
+
+We're going to use both Express and React for this application, so let's start with our folder structure. From the command line, run the following commands:
+
+Make a parent folder called `reactauthapp`:
+
+```
+mkdir reactauthapp
+```
+
+This folder will hold both the front and back end files.
+
+`cd` into that parent folder, and then run:
+```
+npx create-react-app client
+```
+This creates a React app inside the parent directory.
+
+Now, make a `server.js` file in the parent directory with:
+
+```
+touch server.js
+```
+
+This will hold our Express server.
+
+Lastly, in the `reactauthapp` folder, run:
+
+```bash
+npm init -y
+```
+Confirm that a package.json was created in the root `reactauthapp` folder.
+
+At the end, your folder structure should look like this:
+
+```
+reactauthapp
+  client
+  server.js
+  package.json
+```
+
+To cover all of your bases, I recommend `cd`-ing into your `client` folder and running
+
+```
+npm start
+```
+
+to make sure your React app starts up correctly.
+
+
+## Write the Express Server
+
+*"Wait", you may say, "why do I need an Express server? Can't I just use Router?"*
+
+Again, a good question. You need an Express server, because there needs to be somewhere for the access token to be stored that isn't on the browser to pass
```

Comment:
You should mention the authorization code grant above, to prep people for this. You could point folks to the modern guide to oauth, where this grant is discussed in great detail.